builder/chroot-script.sh

```diff
@@ -137,7 +50,7 @@ apt-get install -y \
   raspi-config
 
 # install special Docker enabled kernel
-if [ -z "${KERNEL_URL}" ]; then
+if [[ -z ${KERNEL_URL+x} ]]; then
```

Comment:
Is' nt it better to use "${KERNEL_URL}" as before or why do you write ${KERNEL_URL+x}?

Reply:
I have changed the script to fail on undefined variables (allows for more robust scripts), if I remove the `+x`, the script will fail. This is the right way of checking whether or not a variable is set. [See this ServerFault answer](https://serverfault.com/a/363645/93330)